packages/mask/src/plugins/RedPacket/SNSAdaptor/index.tsx

```diff
@@ -94,7 +94,7 @@ const sns: Plugin.SNSAdaptor.Definition = {
     ApplicationEntries: [
         (() => {
             const icon = <RedPacketIcon />
-            const name = <Trans i18nKey="plugin_red_packet_name" />
+            const name = <Trans ns={PluginId.RedPacket} i18nKey="name" />
```

Comment:
This field support I18NField.

Reply:
ReactNode one looks cleaner, doesn't it?
```diff
-            description: <Trans ns={PluginId.Gitcoin} i18nKey="description" />,
+            description: { i18nKey: `${PluginId.Gitcoin}:description`, fallback: 'description' },
```

In addition, all `ApplicationEntries` for Definition of all plugins are using `<Trans />` instead of `I18NField`, I think it's better to keep them align.

Reply:
If you're not using them by `t.description` or `Translate.description`, please make sure they start with `__`, in this case, `__description`.

We'll have a locale kit in the future to remove unused fields based on the type analysis, starting from `__` is a escape path, "don't remove this field even it is not used in the type level"

Reply:
ping

Reply:
Instead of `__` prefix, can we define some reserved keys, such as `name`, `description`, etc, which can't be removed even they are not used, as each plugin needs to reserve them.


Our i18n keys are all in `snake_case`, we can force the reserved keys in `UpperCamelCase`, and normal keys in `lowerCamelCase` or `snake_case`.

Reply:
yes we can make some name blessed, but we still need a mark if devs want to use their own reserved names. `__` has already baked into the plugin infra

Reply:
TBH, `__` looks weird. 😂

Reply:
Do you have a better suggestion? We can use another prefix/suffix.

Reply:
My suggestion is  using `UpperCamelCase` for reserved keys.

Reply:
Track this issue on #6357.